e2e/cypress/integration/channel/system_message_spec.js

```diff
@@ -73,13 +75,13 @@ describe('System Message', () => {
         cy.getLastPost().then(validateSingle);
 
         // # Update the status to a long string
-        cy.updateChannelHeader('>' + ' newheader'.repeat(20));
+        cy.updateChannelHeader('> ' + newHeader.repeat(20));
 
         // * Check that the status is updated and is spread on more than one line
         cy.getLastPost().
             should('contain', 'System').
             and('contain', '@user-1 updated the channel header from:').
-            and('contain', ' newheader'.repeat(20));
+            and('contain', newHeader.repeat(20));
```

Comment:
For some reasons, the Cypress is referring to the previous post here and trying to verify it's contents. The test failed here because it's expecting the text in the previous postID. Although getLastPost was mentioned, if you check the previous Post is highlighted by Cypress

![image](https://user-images.githubusercontent.com/1429138/62547523-f52ae880-b882-11e9-9865-38f9a44e020f.png)

Reply:
Thanks for testing.  Fix that by using new team.